app/components/category-members-grouped.js

```diff
@@ -30,11 +30,15 @@ export default Component.extend(
 
 				this.get('loadPage')(page)
 					.then(() => {
-						const navHeight = $('.site-head').outerHeight() + $('.site-head-fandom-bar').outerHeight(),
-							scrollTop = this.$().offset().top - navHeight;
+						const navHeight = document.querySelector('.site-head').offsetHeight +
+							document.querySelector('.site-head-fandom-bar').offsetHeight,
+							scrollTop = offset(this.element).top - navHeight;
 
 						run.scheduleOnce('afterRender', this, () => {
-							$('html, body').animate({scrollTop});
+							window.scroll({
```

Comment:
don't we need polyfill for that? http://iamdustan.com/smoothscroll/

Reply:
I'd say not for now, if browser does not support it, it'll just scroll immediately